.github/workflows/ci-unittests.yml

```
@@ -0,0 +1,57 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI Unittests
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  schedule:
+    # Nightly run unit tests to check for implicit dependency issues between pull requests.
+    - cron: '0 0 * * *'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build_test:
+    # Running in containers, so this unknown environment is fine.
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+      # #e want all python versions tested even if one of them happens to fail
```

Typo from the source file carried over:
```suggestion
      # We want all python versions tested even if one of them happens to fail
```